models/lido/liquidity/linea/lido_liquidity_linea_syncswap_pools.sql

```diff
@@ -0,0 +1,291 @@
+{{ config(
+    schema='lido_liquidity_linea',
+    alias = 'syncswap_pools',
+    materialized = 'incremental',
+    file_format = 'delta',
+    incremental_strategy = 'merge',
+    unique_key = ['pool', 'time'],
+    post_hook='{{ expose_spells(\'["linea"]\',
+                                "project",
+                                "lido_liquidity",
+                                \'["pipistrella"]\') }}'
+    )
+}}
+
+{% set project_start_date = '2023-11-01' %}
+
+with  pools as (
+select pool AS address,
+      'linea' AS blockchain,
+      'syncswap' AS project,
+      *
+from {{source('syncswap_linea','SyncSwapClassicPoolFactory_evt_PoolCreated')}}
+where (token0 = 0xB5beDd42000b71FddE22D3eE8a79Bd49A568fC8F
+      OR token1 = 0xB5beDd42000b71FddE22D3eE8a79Bd49A568fC8F)
+)      
+
+, tokens as (
+ select distinct token,
+         case when token = 0xb5bedd42000b71fdde22d3ee8a79bd49a568fc8f then 0x7f39c581f595b53c5cb19bd0b3f8da6c935e2ca0
+             when token = 0x176211869ca2b568f2a7d4ee941e073a821ee1ff then 0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48
+             when token = 0xa219439258ca9da29e9cc4ce5596924745e12b93 then 0xdac17f958d2ee523a2206206994597c13d831ec7
+             when token = 0xe5d7c2a44ffddf6b295a15c148167daaaf5cf34f then 0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2
+        end as token_eth     
+ from (
+ select token0 as token
+ from {{source('syncswap_linea','SyncSwapClassicPoolFactory_evt_PoolCreated')}}
+ where token1 = 0xB5beDd42000b71FddE22D3eE8a79Bd49A568fC8F
+ union all
+ select token1
+ from {{source('syncswap_linea','SyncSwapClassicPoolFactory_evt_PoolCreated')}}
+ where token0 = 0xB5beDd42000b71FddE22D3eE8a79Bd49A568fC8F
+ union all
+ select 0xB5beDd42000b71FddE22D3eE8a79Bd49A568fC8F
+ ) t
+ )
+
+ , tokens_prices_daily as (     
+select distinct 
+      DATE_TRUNC('day', minute) AS time,
+      t.token AS token,
+      symbol,
+      decimals,
+      AVG(price) AS price
+FROM {{source('prices','usd')}} p
+  JOIN tokens t on p.contract_address = t.token_eth
+{% if not is_incremental() %}
+WHERE DATE_TRUNC('day', p.minute) >= DATE '{{ project_start_date }}'
+{% else %}
+WHERE DATE_TRUNC('day', p.minute) >= DATE_TRUNC('day', NOW() - INTERVAL '1' day)
```

Comment:
We may use the `incremental_predicate` for all incremental filters:
```suggestion
WHERE {{incremental_predicate('p.minute')}}
```